aws_lambda_powertools/shared/headers_serializer.py

```diff
@@ -65,8 +66,9 @@ def serialize(self, headers: Dict[str, Union[str, List[str]]], cookies: List[Coo
             if isinstance(values, str):
                 payload[key].append(values)
             else:
-                for value in values:
-                    payload[key].append(value)
+                if values:
```

Comment:
when you accept a falsy value but not null, check for null otherwise a `X-Is-Admin: false` will be omitted here. For example, `if values is None`, would work in both cases.